lightning/src/ln/channelmanager.rs

```diff
@@ -2397,6 +2460,11 @@ impl<Signer: Sign, M: Deref, T: Deref, K: Deref, F: Deref, L: Deref> ChannelMana
 		self.our_network_pubkey.clone()
 	}
 
+	/// Gets the node secret held by this ChannelManager
+	pub fn get_our_node_secret(&self) -> SecretKey {
```

Comment:
Because its a secret key, we should probably return a reference to it instead of copying the actual secret bytes in memory. Likely to be optimized into the same thing, but its still "best practice".